modules/mvt/test/data/fetch_data.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/local/bin/python3
+
+"""Simple script to fetch MVT tiles for testing"""
+import requests
+import os
+
+TABLE = 'cartodb-gcp-backend-data-team.alasarr'
+
+DATASETS = {
+    'blockgroup': f'{TABLE}.usa_blockgroup_population',
+    'blockgroup_numprop': f'{TABLE}.usa_blockgroup_tileset_numprop',
+    'census_numprop': f'{TABLE}.usa_censustract_2015_tileset_numprop',
+    'zipcodes': f'{TABLE}.usa_zcta_2015_tileset',
+    'zipcodes_numprop': f'{TABLE}.usa_zcta_2015_tileset_numprop',
+    'counties_numprop': f'{TABLE}.usa_county_2015_tileset_numprop',
+}
+
+
+SERVER = 'https://maps-api-v2.us.carto.com/user/aasuero/bigquery'
+
+TILES = [
+    {'z': 4, 'x': 4, 'y': 6},
+    {'z': 4, 'x': 3, 'y': 6},
+    {'z': 5, 'x': 8, 'y': 12},
+    {'z': 6, 'x': 18, 'y': 24},
+    {'z': 7, 'x': 35, 'y': 50},
+    {'z': 8, 'x': 75, 'y': 96}
+]
+for name, dataset in DATASETS.items():
```

Comment:
So basically this script is downloading 36 tiles? How big in total is this data? Is it too big to just store in this repo? Is the marginal benefit of each additional xyz index large? Otherwise we could store just 6 tiles locally; one tile for each dataset.

Reply:
The files chosen are large, so in total 31MB. I don't think it is worth storing them in the repo. However I think it is good to keep the script around in case we want to debug on some tiles that cause problems in the future & or we want to check performance against. As noted elsewhere, this script is not run as part of CI, that uses already checked in mvt test data to run sanity checks, which should guard against regressions